src/main/java/com/launchdarkly/sdk/server/integrations/DynamoDbDataStoreBuilder.java

```diff
@@ -19,22 +21,52 @@
 /**
  * Builder/factory class for the DynamoDB data store.
  * <p>
- * Obtain an instance of this class by calling {@link com.launchdarkly.sdk.server.integrations.DynamoDb#dataStore(String)}.
- * After calling its methods to specify any desired custom settings, wrap it in a
- * {@link com.launchdarkly.sdk.server.integrations.PersistentDataStoreBuilder}
- * by calling {@code Components.persistentDataStore()}, then pass the result into the SDK configuration with
- * {@link com.launchdarkly.sdk.server.LDConfig.Builder#dataStore(com.launchdarkly.sdk.server.interfaces.DataStoreFactory)}.
- * You do not need to call {@link #createPersistentDataStore(ClientContext)} yourself to build the actual data store; that
- * will be done by the SDK.
- * <p>
+ * This can be used either for the main data store that holds feature flag data, or for the Big
+ * Segment store, or both. If you are using both, they do not have to have the same parameters. For
+ * instance, in this example the main data store uses a table called "table1" and the Big Segment
+ * store uses a table called "table2":
+ *
+ * <pre><code>
+ *     LDConfig config = new LDConfig.Builder()
+ *         .dataStore(
+ *             Components.persistentDataStore(
+ *                 DynamoDb.dataStore("table1")
+ *             )
+ *         )
+ *         .bigSegments(
+ *             Components.bigSegments(
+ *                 DynamoDb.dataStore("table2")
+ *             )
+ *         )
+ *         .build();
+ * </code></pre>
+ *
+ * Note that the builder is passed to one of two methods,
+ * {@link com.launchdarkly.sdk.server.Components#persistentDataStore(PersistentDataStoreFactory)} or
+ * {@link com.launchdarkly.sdk.server.Components#bigSegments(BigSegmentStoreFactory)}, depending on
+ * the context in which it is being used. This is because each of those contexts has its own
+ * additional configuration options that are unrelated to the DynamoDb options. For instance, the
+ * {@link com.launchdarkly.sdk.server.Components#persistentDataStore(PersistentDataStoreFactory)}
+ * builder has options for caching:
+ *
+ * <pre><code>
+ *     LDConfig config = new LDConfig.Builder()
+ *         .dataStore(
+ *             Components.persistentDataStore(
+ *                 DynamoDb.dataStore("table1")
+ *             ).cacheSeconds(15)
+ *         )
+ *         .build();
+ * </code></pre>
+ *
```

Comment:
I know you're following a pattern that I started, but... I'm not sure we really need to be duplicating the exact same 40-line comment in two places like this. It makes it easy for us to update text in one place and forget to update it in another, and personally as a developer if I see something like this I feel compelled to read through both versions thinking maybe there's some difference between them that I'm missing.

So, I'd kind of like to pick one. But I'm not sure which one. I'm leaning toward saying it should be in `DynamoDb.dataStore()`, because that's the entry point, and that `DynamoDbDataStoreBuilder` should have a simpler comment explaining that this is the configuration builder and that you can see `DynamoDb.dataStore()` for more information. I would keep the last paragraph ("The AWS SDK provides..."), though.